linchpin/ansible_runner.py

```diff
@@ -82,6 +82,28 @@ def subprocess_runner(cmd, shell=False):
     return proc.returncode, stdout, stderr
 
 
+def ansible_runner_2x(playbook_path,
+                      extra_vars,
+                      options=None,
+                      inventory_src='localhost',
```

Comment:
@JunqiZhang0  Why is ansible runner changed in this PR?
I think it is reverted back. 
could you drop the commit?